qutebrowser/browser/qutescheme.py

```diff
@@ -268,6 +269,13 @@ def qute_pyeval(_url):
     return 'text/html', html
 
 
+@add_handler('spawn_output')
```

Comment:
This *might* work, but having underlines in URLs looks a bit weird 😆 I'd prefer `spawn-output` here.